db/migrate/20230503112422_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author foreign_key: {to_table: 'users'}
+      t.references :post foreign_key: true
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ Required ] I think it would be better if you fix some syntax errors on this file when I run the migrations I get the error below. I would like to suggest that you refactor the code to the one below.

```ruby
class CreateLikes < ActiveRecord::Migration[7.0]
  def change
    create_table :likes do |t|
      t.references :author, foreign_key: {to_table: 'users'}
      t.references :post, foreign_key: true
      t.timestamps
    end
  end
end
```

## Error when running the migrations
![Screenshot from 2023-05-04 02-34-40](https://user-images.githubusercontent.com/16392046/236073591-a7f5b9d3-0906-4639-900d-c6373949b758.png)